test/parallel/test-os.js

```diff
@@ -116,7 +116,8 @@ const interfaces = os.networkInterfaces();
 switch (platform) {
   case 'linux':
   {
-    const filter = (e) => e.address === '127.0.0.1';
+    const filter =
+      (e) => e.address === '127.0.0.1' && e.netmask === '255.0.0.0';
     const actual = interfaces.lo.filter(filter);
     const expected = [{ address: '127.0.0.1', netmask: '255.0.0.0',
```

Comment:
Some reasons to maybe not do that:

* If we want to add macOS to the switch statement (in another PR), it will have a different `expected` value. (The MAC address is different. Yes, the MAC address is different on Mac. Ha ha, ha ha, gosh, I'm funny.)

* Probably don't want to replace the `switch` because we need a do-nothing `default` for macos, smartos... A switch makes sense here, I think. I mean, you can make it work with an `if`, of course. But I think this more verbose code is actually more readable even though it does violate DRY. (I often don't value DRY much in tests, TBH.)

Even if I'm wrong about that evaluation... must...resist....scope creep.